tuned_lens/scripts/lens.py

```diff
@@ -119,10 +121,17 @@ def main(args):
         model.to(local_rank)
 
     # Load tokenizer & data
-    tokenizer = AutoTokenizer.from_pretrained(
+    if args.tokenizer_class:
```

Comment:
There is a [`tokenizer_type`](https://huggingface.co/docs/transformers/v4.27.2/en/model_doc/auto#transformers.AutoTokenizer.from_pretrained.tokenizer_type) kwarg for `AutoTokenizer.from_pretrained` if possible I think we should use it instead.

Reply:
Ok can you implement that? Sorry I'm out of the country without a laptop for the next week